src/main/kotlin/no/fdk/dataset_catalog/rdf/Namespace.kt

```diff
@@ -102,4 +91,41 @@ class PROV {
         val qualifiedAttribution = model.createProperty("${uri}qualifiedAttribution")
 
     }
-}
\ No newline at end of file
+}
+
+class ExtendedXSD {
+    companion object {
+        private val model = ModelFactory.createDefaultModel()
+        val seeAlso = model.createProperty("${XSD.NS}seeAlso")
```

Comment:
Kan bruke ResourceFactory direkte her:

```suggestion
        val seeAlso = ResourceFactory.createProperty("${XSD.NS}seeAlso")
```

Reply:
0d499fd